TRAMS-API/Controllers/TrustsController.cs

```diff
@@ -26,23 +26,29 @@ namespace TRAMS_API.Controllers
     public class TrustsController : ControllerBase
     {
         private readonly ILogger<TrustsController> _logger;
-        private readonly TrustRepository _trustRepostiory;
-        private readonly IMapper<GetTrustD365Model, GetTrustsModel> _mapper;
+        private readonly TrustsRepository _trustRepostiory;
+        private readonly AcademiesRepository _academiesRepository;
+        private readonly IMapper<GetTrustsD365Model, GetTrustsModel> _getTrustMapper;
+        private readonly IMapper<GetAcademiesD365Model, GetAcademiesModel> _getAcademiesMapper;
 
-        public TrustsController(ILogger<TrustsController> logger, 
+        public TrustsController(ILogger<TrustsController> logger,
                                          IConfiguration config,
-                                         TrustRepository trustRepostiory,
-                                         IMapper<GetTrustD365Model, GetTrustsModel> mapper)
+                                         TrustsRepository trustRepostiory,
+                                         AcademiesRepository academiesRepository,
```

Comment:
Suggestion: Break these out into an interface so that it can be mocked or switched for another implementation in the future.